### ISSUE TITLE
[Java] Use gradle for spring-framework

### ISSUE THREAD

**Issue by waghanza** [the-benchmarker/web-frameworks#3103]
Hi,

Seems that `gradle` has been the default tool when using `spring-framework`.

Is this affirmation correct @sbrannen @rstoyanchev @jhoeller ?

I suggest we use it here.

Regards,

**Comment by sbrannen** [comment]
Yes, the core Spring Framework project has used Gradle for its builds for several years.

**Comment by waghanza** [comment]
Thanks for confirmation, I (or anyone wanted to) will change this. The first contributor use `maven`, but the idea is to use what is elected by frameworks community 😉

**Comment by waghanza** [comment]
I imagine this is also the same for spring-boot

**Comment by sbrannen** [comment]
Spring Boot historically used Maven for its build; however, the Spring Boot team recently migrated to Gradle.

See the [Migrating Spring Boot's Build to Gradle](https://spring.io/blog/2020/06/08/migrating-spring-boot-s-build-to-gradle) blog for details.
